src/main/java/org/codehaus/plexus/util/NioFiles.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Encapsulates use of java7 features, exposing mostly backward compatible types
  */
-@SuppressWarnings( "Since15" )
+@Deprecated
```

Comment:
Provide a pointer to a replacement.  Simply indicating, `replace with direct NIO calls` would be fine, but it's always a pain when you run into a deprecated method and you need to go through the GitHub history to understand why it has been deprecated and what to use instead.